blueprints/modular/contact.yaml

```
@@ -0,0 +1,39 @@
+title: Contact
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        features:
+          type: tab
+          title: Contact
+          fields:
+            header.show_form:
+              type: toggle
+              label: Show Contact Form
+              highlight: 1
+              help: This will select whether to show the form defined in the form par
+              options:
+                1: Yes
+                0: No
+              validate:
+                type: bool
+
+            
+            header.button:
+              type: section
+              title: Contact Button
+              fields:
+                header.button.link:
+                  type: text
+                  label: Mailto Link

```

I want this to be generic so someone could link to Twitter, FB, or whatever else they want.
```suggestion
                  label: Link
```

---

Ah right, sure